ntfy/cli.py

```diff
@@ -44,16 +46,58 @@ def run_cmd(args):
         *map(int, divmod(time() - start_time, 60))
     )
 
+emojis_hash = {}
+try:
+    emojis_json_path = join(dirname(getfile(run_cmd)),
```

Comment:
Given the [implementation](https://github.com/python/cpython/blob/master/Lib/inspect.py#L602-L625) of inspect, I see no reason not to just use `__file__`. AFAICT inspect.getfile is more for finding the file where an unknown object (Is it a module? Is it a Class?)